### PR TITLE
PR: Team page content

### LINKED ISSUE #56: Team page layout & contents
The 2-D grid that was a previous version of the Projects page would be nice for the Team page. I'm thinking for each listed person there should be: full name, a picture (or GitHub logo), and names of projects. Perhaps project names should say "maintainer" or "contributor".

The main criterion to be listed is that you should be a maintainer or contributor (ongoing contributions, not a one-off PR) to a project listed on the Projects page. Also, if you want to be listed on this page as part of the Labs Team, there's an expectation of making contributions to help shape Labs - this could be help maintaining the website, writing a blog post once every couple of months, or other equivalent/useful activities (more detail TBD). This is a sort of "co-op model" (thanks @saulshanabrook for that suggestion).

People to be listed (note, this is 1-2 months old, if someone is missing please comment or ping me on Slack):

Ralf Gommers
Saul Shanabrook
Hameer Abbasi
Carlos Cordoba 
Gonzalo Peña-Castellanos
Edgar Margffoy
Stephannie Jiménez Gacha
Daniel Althviz
Melissa Weber Mendonça
Mars Lee
Yuvi Panda
Anthony Scopatz 
Athan Reines
Aaron Meurer
Chris Ostrouchov
James Bourbeau
Ivan Ogasawara
Peter Bell
Pearu Peterson
Sameer Deshmukh
Nathan Goldbaum
Guilherme Leobas
Anirrudh Krishnan
Marc Garcia

### PR DESCRIPTION
**Pull request by dalthviz**
### TODO
- [x] Check the correctness of the content and missing data (photos, names, GitHub profile, projects).
- [x] Check if more people needs to be added.

A preview:

image

Fixes #56

### CODE REVIEW

**Comment by rgommers** [comment]
Let's ping people here so they can review their own entry (I'll add what I know):

- [ ] @hameerabbasi: add NumPy
- [ ] 
- [ ] 
- [ ] 
- [ ] 
- [ ]

**Comment by rgommers** [comment]
Let's ping people here so they can review their own entry (I'll add what I know). 

To everyone: can you please review this list, and confirm you want to be on it? Being on this Team page says you are "Labs affiliated", which implies you are actively contributing to or maintaining at least one Labs-supported project (whether as part of your Quansight time or not), and you are happy to commit 2-4 hrs a month to helping build Labs (i.e. write a blog once in while, help improve the website, review the annual report, etc.). 

Please tick your own box in this comment if you've reviewed and want to be on this page.

- [ ] @hameerabbasi: add NumPy
- [ ] @saulshanabrook: ok
- [ ] @ccordoba12: ok
- [ ] @goanpeca: ok
- [ ] @andfoy: ok
- [ ] @steff456: ok 
- [ ] @dalthviz: ok
- [ ] @melissawm: ok
- [ ] @MarsBarLee: ok
- [ ] @yuvipanda: ok
- [ ] @scopatz: add PyTables
- [ ] @kgryte: add `stdlib.js`
- [ ] @asmeurer: add Numba
- [ ] @costrouc: add python-api-inspect
- [ ] @jrbourbeau: add Numba
- [ ] @xmnlab: ok
- [ ] @peterbell10: add Uarray
- [ ] @pearu: add Numba
- [ ] @v0dro: ok
- [ ] please remove Nathan, he unfortunately left us
- [ ] @guilhermeleobas: ok
- [ ] @anirrudh: not sure if JupyterLab is right - Ani did you contribute?
- [ ] @datapythonista: ok

**Comment by melissawm** [comment]
I can't tick the box but LGTM.

**Comment by andfoy** [comment]
I can't tick, but I agree

**Comment by steff456** [comment]
I can’t tick if neither, but I agree

**Comment by guilhermeleobas** [comment]
I can't tick, but I agree [3]

**Comment by hameerabbasi** [comment]
`Pydata/Sparse` -> `PyData/Sparse`, note the capitalization. Other than that, all good.

**Comment by kgryte** [comment]
Agree.

**Comment by asmeurer** [comment]
Is there a reason we aren't all members of the GitHub org? That's why we can't tick the boxes. It's also impossible for me to route Quansight Labs notifications to my Quansight email unless I am a member.

**Comment by asmeurer** [comment]
My data looks fine. I would suggest sorting people alphabetically by last name.

**Comment by anirrudh** [comment]
@rgommers nope, not yet. Just a couple issues closed.

**Comment by pearu** [comment]
Pearu: Add rbc.

**Comment by rgommers** [comment]
@dalthviz I pinged the six missing approvals on Slack. Let's give it another 1-2 days.

**Comment by goanpeca** [comment]
@dalthviz @rgommers.  Question:

Projects listed under my profile are:

1) general open source projects or 

If this then it is ok

2) open source projects under the Labs umbrella?

If this then please add [beeware](https://beeware.org/) and [lektor](https://www.getlektor.com/)

Thanks!

**Comment by rgommers** [comment]
@goanpeca good question. from my message above:

> .... implies you are actively contributing to or maintaining at least one Labs-supported project (whether as part of your Quansight time or not)

So I'd say we want to list projects that are related to Labs, and we want to list on the Projects page so we can explain there in one paragraph what we contribute to those projects. So I'd suggest it's okay as is for you.

File: galleries/team/metadata.yml [hunk 1]
```diff
@@ -1,31 +1,111 @@
...
 ---
-name: gonzalo_peña.png
-title: Gonzalo Peña-Castellanos
-github: goanpeca
+name: sameer_deshmukh.png
+title: Sameer Deshmukh
+github: v0dro
+projects: ['XND']
+---
+name: marc_garcia.png
```
Comment (datapythonista) [inline galleries/team/metadata.yml:32]: I sent Ashley a more updated picture for the website. Not sure where did you get this from, but probably better to use the one I sent.

File: galleries/team/metadata.yml [hunk 2]
```diff
@@ -1,31 +1,111 @@
...
+title: Marc Garcia
+github: datapythonista
+projects: ['Ibis', 'Pandas']
```
Comment (datapythonista) [inline galleries/team/metadata.yml:35]:
```suggestion
projects: ['Ibis', 'pandas']
```

File: galleries/team/metadata.yml (hunk 1 above)
Comment (dalthviz) [reply galleries/team/metadata.yml:32]: I used for the pictures the ones available on the bambooHR website. I will ask Ashley for the one you sent her :+1:

**Comment by rgommers** [comment]
@dalthviz I think we're about complete here. We just gained two new people at the start of April: @juanis2112 and @mattip. Could you add them? Their projects:
- Juanita: Spyder
- Matti: NumPy, PyPy

**Comment by mattip** [comment]
I can’t tick but I agree

**Comment by v0dro** [comment]
Just need a change with the photo.

**Comment by rgommers** [comment]
Okay let's call this good:) Thanks @dalthviz!

**Pull request merged**